### PR TITLE
Added previous Message support

### PR DESCRIPTION
**Pull request by burdoto**

### CODE REVIEW

**Comment by burdoto** [comment]
issue paused due to https://github.com/Javacord/Javacord/issues/515

File: src/main/java/de/comroid/util/ChannelUtils.java [hunk 1]
```diff
@@ -43,4 +46,13 @@
                     updater.update().exceptionally(ExceptionLogger.get());
                 });
     }
+
+    public static Optional<Message> previous(Message before) {
+        return before.getChannel()
```
Comment (Vampire) [inline src/main/java/de/comroid/util/ChannelUtils.java]:
```suggestion
        return before.getMessagesBefore(1);
```

Comment (peanutbother) [reply src/main/java/de/comroid/util/ChannelUtils.java]: Thanks for taking your time to also comment here with a fix suggestion!

Comment (Vampire) [reply src/main/java/de/comroid/util/ChannelUtils.java]: It is actually just a pointer, as it returns a completable future, not an optional :-)

**Pull request merged**